.env.mainnet

```diff
@@ -76,10 +76,10 @@ L2_ETHERSCAN_API_KEY=
 # *************** PRIVATE TESTNET ***************
 
 # RPC URL of the L1 network from which a private L1 test network is forked if a private test network is used, e.g. Infura, Alchemy
-L1_FORK_RPC_URL=https://sepolia.infura.io/v3/YOUR_INFURA_KEY
+L1_FORK_RPC_URL=https://ethereum-rpc.publicnode.com
 
 # RPC URL of the L2 network from which a private L2 test network is forked if a private test network is used, e.g. Infura, Alchemy
-L2_FORK_RPC_URL=https://optimism-sepolia.infura.io/v3/YOUR_INFURA_KEY
+L2_FORK_RPC_URL=https://rpc.api.lisk.com
```

Comment:
Is `.env.mainnet` even intended to be used to create a private testnet? Shouldn't one use `.env.devnet` for it? Therefore, wouldn't it be better to have no value here? Same for `L1_FORK_RPC_URL`.

Reply:
This is the mainnet env file, so we should provide values relevant for mainnet deployments